paddle/fluid/common_flags.cc

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "gflags/gflags.h"
+
+DEFINE_bool(use_mkldnn, false, "Use MKLDNN to run");
```

Comment:
Why not put into `paddle/fluid/platform/flags.cc`?

Reply:
The idea looked nice but the MAC CIs (MAC_PR_CI, MAC_PR_CI_python35) and  PR_CI_Py35 found the following error:

> [15:45:03]W:	 [Step 1/1] Undefined symbols for architecture x86_64:
[15:45:03]W:	 [Step 1/1]   "fLB::FLAGS_use_mkldnn", referenced from:
[15:45:03]W:	 [Step 1/1] ld: symbol(s) not found for architecture x86_64
[15:45:03]W:	 [Step 1/1] clang: error: linker command failed with exit code 1 (use -v to see invocation)
logs: [PR_CI_Py35](https://paddle-docker-tar.cdn.bcebos.com/buildLog/379449.log) [PaddlePaddle_MAC_CI](https://paddle-docker-tar.cdn.bcebos.com/buildLog/379450.log) [PaddlePaddle_MAC_CI](https://paddle-docker-tar.cdn.bcebos.com/buildLog/379449.log)

So I've brought back the original idea to put 'use_mkldnn' at the top level CMake to be accessible for platform and imperative as well.